### ISSUE TITLE
DietPi-Banner | Large Format Hostname

### ISSUE THREAD

**Issue by matellis** [MichaIng/DietPi#5110]
#### Is your feature request related to a problem? Please describe:

- I can get confused which node I'm logged into when I'm doing repetitive tasks across several or more nodes
- For instance, if I'm making some changes to one instance, and then want to port them to the others, it's very easy to get lost which server I'm logged into

#### Describe the solution you'd like:

- It would be nice if I could put this at the top of the screen, before the start of the banner, like this:

![Screencap](https://imgur.com/NbuI7Wu.png)

#### Describe alternatives you've considered:

- Today, I use `figlet` to show the hostname on login, via .bashrc but this prints out the name after the banner and it's hard to parse the banner

**Comment by MichaIng** [comment]
Done with: #5113
Changelog: https://github.com/MichaIng/DietPi/commit/e82e6ef
